app/agent/rest/v1/agent.js

```diff
@@ -0,0 +1,10 @@
+angular.module('portainer.agent')
```

Comment:
As this endpoint did not actually change between v1/v2, I wonder if we can simply keep using `Agent` for both v1 and v2 instead of `AgentVersion1` and `Agent` ?

Reply:
I think we should use the new service, because we will need this change anyway one day, so why not do it now when we're on it?

Reply:
IMO we should only "override" the API features that are different across versions. If we don't it's going to be super unclear by having a look at the codebase what's actually in use.

Or we go in full way, by creating a v2 folder as well and overriding for each. I think that simply using the same rest service with the version injected via  `StateManager.getAgentApiVersion` is a neat approach to begin with.

Reply:
Ignore my comment, we need to keep this one because the current version of the agent do not support the version in the URL path.